user_manual/android_app.rst

```diff
@@ -305,6 +341,26 @@ Tap it and enter the name of the new folder in the *Folder name* dialog.
 Then, tap the newly created folder and tap *Choose* in the bottom right-hand corner. 
 You'll see that the path has been updated.
 
+.. _upload_pictures_directly_from_the_camera_label:
+
+Upload Pictures Directly From The Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.7.0
+
+.. figure:: images/camera/share-from-camera-owncloud-android-app.jpg
```

Comment:
I think that **Upload pictures directly from the camera** feature would fit better in **Files** section than in **Settings** one, below or even inside **Creating New Content** subsection

Reply:
Ok.